### PR TITLE
Fixed documentation for Free Node paradigm

### PR DESCRIPTION
**Pull request by Thenerdstation**
Tested it manually and it looks fine.

### CODE REVIEW

File: tensornetwork/network_components.py [hunk 1]
```diff
@@ -1476,11 +1476,12 @@ def _contract_trace(edge: Edge, name: Optional[Text] = None) -> BaseNode:
...
-  Contract an edge connecting two nodes.
+  """Contract an edge connecting two nodes.
+
   All edges of `node1` and `node2` are passed on to the new node,
   and `node1` and `node2` get a new set of dangling edges.
-  `edge is disabled before returning.
+  `edge` is disabled before returning.
+
   Args:
     edge: The edge contract next.
```
Comment (sleichen) [inline tensornetwork/network_components.py]: Should this just read "The edge to contract."?

File: tensornetwork/network_operations.py [hunk 2]
```diff
@@ -92,14 +93,14 @@ def transpose(node: BaseNode,
 
 def copy(nodes: Iterable[BaseNode],
          conjugate: bool = False) -> Tuple[dict, dict]:
-  """
+  """Return a copy of the given nodes.
```
Comment (sleichen) [inline tensornetwork/network_operations.py]: I would be more specific in this docstring. Maybe say "Copy the given nodes and their edges, returning a dictionary linking originals to copies."

File: tensornetwork/network_operations.py [hunk 3]
```diff
@@ -92,14 +93,14 @@ def transpose(node: BaseNode,
...
 
-  Return a copy of the TensorNetwork.
   Args:
     nodes: An `Iterable` (Usually a `List` or `Set`) of `Nodes`.
     conjugate: Boolean. Whether to conjugate all of the nodes in the
-      `TensorNetwork` (useful for calculating norms and reduced density
-      matrices).
+        `TensorNetwork` (useful for calculating norms and reduced density
```
Comment (sleichen) [inline tensornetwork/network_operations.py]: Remove reference to `TensorNetwork`.

File: tensornetwork/network_operations.py [hunk 4]
```diff
@@ -92,14 +93,14 @@ def transpose(node: BaseNode,
...
+        matrices).
+
   Returns:
     A tuple containing:
       node_dict: A dictionary mapping the nodes of the original
```
Comment (sleichen) [inline tensornetwork/network_operations.py]: Change wording to avoid use of "network".

File: tensornetwork/network_components.py (hunk 1 above)
Comment (Thenerdstation) [reply tensornetwork/network_components.py]: Yes. Done

File: tensornetwork/network_operations.py (hunk 2 above)
Comment (Thenerdstation) [reply tensornetwork/network_operations.py]: Done.

File: tensornetwork/network_operations.py (hunk 3 above)
Comment (Thenerdstation) [reply tensornetwork/network_operations.py]: Done.

File: tensornetwork/network_operations.py (hunk 4 above)
Comment (Thenerdstation) [reply tensornetwork/network_operations.py]: Done

**Pull request merged**